.github/workflows/docker-image-test.yml

```diff
@@ -12,15 +12,12 @@ jobs:
             - name: Checkout PR branch
               uses: actions/checkout@v2
 
-            - name: Set up QEMU
-              uses: docker/setup-qemu-action@v1
-
-            - name: Set up Docker Buildx
-              uses: docker/setup-buildx-action@v1
+            - name: Set up Depot CLI
+              uses: depot/setup-action@v1
 
             - name: Build
               id: docker_build
-              uses: docker/build-push-action@v2
+              uses: depot/build-push-action@v1
               with:
-                  push: false
```

Comment:
`push: false` is the default (for both Depot's and Docker's `build-push-action`)